src/main/scala/com/azavea/modellab/op/MapRanges.scala

```diff
@@ -0,0 +1,39 @@
+package com.azavea.modellab.op
+
+import geotrellis.raster._
+import geotrellis.spark._
+
+case class MapRanges(
+  input: Op,
+  mappings: Seq[((Double, Double), Option[Double])]
+) extends Op {
+  def calc(zoom: Int, bounds: GridBounds) = {
+    val rasterRDD = input(zoom, bounds)
+    val metaData = rasterRDD.metaData
+    // Construct a list of partial functions to be chained together
+    val partialFuncs: Seq[PartialFunction[Double, Double]] =
+      mappings map { case (range: (Double, Double), mapped: Option[Double]) =>
```

Comment:
Does this serialize? I would expect this serialization to chain partialFuncs -> mappings -> MapRanges Instance -> rest of the ops.